src/photos.cs

```diff
@@ -371,17 +380,19 @@ interface PHAssetResourceManager
 		void CancelDataRequest (int requestID);
 	}
 
+	delegate void PHAssetResourceProgressHandler (double progress);
+
 	[iOS (9,0)]
 	[TV (10,0)]
-	[NoMac]
-	[BaseType (typeof(NSObject))]
-	interface PHAssetResourceRequestOptions : NSCopying
-	{
+	[Mac (10,15, onlyOn64 : true)]
+	[BaseType (typeof (NSObject))]
+	interface PHAssetResourceRequestOptions : NSCopying {
+
 		[Export ("networkAccessAllowed")]
 		bool NetworkAccessAllowed { [Bind ("isNetworkAccessAllowed")] get; set; }
 
 		[NullAllowed, Export ("progressHandler", ArgumentSemantic.Copy)]
-		Action<double> ProgressHandler { get; set; }
+		PHAssetResourceProgressHandler ProgressHandler { get; set; }
```

Comment:
I'm fairly certain this is a breaking change (the api diff will tell for sure).

Reply:
Okay I will change this and the other breaking change. I am not sure why I even changed the original